core/dbt/adapters/base/connections.py

```diff
@@ -400,15 +400,36 @@ def _add_query_comment(self, sql: str) -> str:
 
     @abc.abstractmethod
     def execute(
-        self, sql: str, auto_begin: bool = False, fetch: bool = False
+        self, sql: str, auto_begin: bool = False, fetch: bool = False, limit: Optional[int] = None
```

Comment:
This was leftover from the implementation of limit in a previous change. The type here matches that of the original change.